ValueSet/NHSDigital-DocumentType.xml

```diff
@@ -1,7873 +0,0 @@
-<ValueSet xmlns="http://hl7.org/fhir">
```

Comment:
What is the rationale behind deleting the ValueSet, is it no longer used/replaced by UKCore etc.?

Reply:
Yes it's a duplicate of the UKCore valueSet